src/mon/MDSMonitor.cc

```diff
@@ -342,7 +342,7 @@ bool MDSMonitor::preprocess_beacon(MonOpRequestRef op)
     goto ignore;
   }
 
-  dout(12) << "preprocess_beacon " << *m
+  dout(7)  << "preprocess_beacon " << *m
```

Comment:
huh, can we bump this to 10 instead? or lower it down to 5? I get the idea of having it at an intermediate level, but it makes it harder to standardize the debug levels. And people do tend to rely a lot more on 0, 1, 5, 10, and this will end up showing with a level 10 anyway.